udata/core/dataset/api.py

```diff
@@ -59,16 +60,28 @@
 from .forms import (
     ResourceForm, DatasetForm, CommunityResourceForm, ResourcesListForm
 )
-from .search import DatasetSearch
 from .exceptions import (
     SchemasCatalogNotFoundException, SchemasCacheUnavailableException
 )
 from .rdf import dataset_to_rdf
 
+
+class DatasetApiParser(ModelApiParser):
+    sorts = {
```

Comment:
I think it's not exactly the same sorts parameters as in the `class DatasetSearch`. Shouldn't we be iso?

Reply:
We should probably be ,iso for sorting to work in admin (sorting datasets by columns in /fr/admin/site/ for example).